apps/kita/app/Http/Controllers/Admin/AdminUserController.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace app\Http\Controllers\Admin;
+
+use App\Http\Controllers\Controller;
+use App\Models\AdminUser;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Facades\Validator;
+use Illuminate\Validation\Rule;
+use Illuminate\Support\Facades\Auth;
+use App\Consts\CommonConst;
+
+class AdminUserController extends Controller
+{
+    /**
+     * 管理者画面一覧と検索
+     * @param Request $request
+     * @return \Illuminate\Contracts\Foundation\Application|\Illuminate\Contracts\View\Factory|\Illuminate\Contracts\View\View
+     */
+    public function index(Request $request) {
```

Comment:
https://docs.google.com/spreadsheets/d/17MfQxuTdDpz9DM9tNFJFKhZ0PFHEw1bul_AkMO5eoF0/edit#gid=1489262498

↑ こちら確認してますか？

PSR2のコーディング規約に沿ってくださいね。
```suggestion
    public function index(Request $request) 
    {
```

Reply:
あとインデント確認してください。